src/BuildScriptGenerator/DefaultRunScriptGenerator.cs

```diff
@@ -123,6 +123,12 @@ public string GenerateBashScript(RunScriptGeneratorContext ctx)
 #pragma warning restore CA2201
             }
 
+            BuildConfigurationFIle buildConfigFile = BuildConfigurationFIle.Create(ctx.SourceRepo.ReadFile("appsvc.yaml"));
```

Comment:
Would there be a way to inject this through `create-script`? Here it feels strange to just append at the end of the file a new command without integrating this with the rest of the script generation. Would there be a way to reconcile these?

Reply:
When talked about this with Tulika we concluded to append the command at the end of run.sh. But your suggestion makes sense. I will check if we can inject the user command through `create-script`.

Reply:
Injecting via `create-script`: If the goal is to take `appsvc.yaml` code and place it into `run.sh` then it would beneficial for Oryx to add the code into `run.sh`. Injecting the code through `create-script` would make sense if we were not reading from `appsvc.yaml`

Appending `appsvc.yaml` to the end of `run.sh`: Adding to `DefaultRunScriptGenerator.cs` adds this to all platforms, not just DotNet. Is this what we want to do? If so, keep in mind that python can run in virtual environment an non-virtual environment which means appending this to the end may not work. Other platforms might have similar "gotchas" so we need to look inside the run.sh and place the `appsvc.yaml` content in the correct order. If you provide a sample `run.sh` we can look at it together. I would suggest revisiting appending this for all platforms. 

Side note: Here is the source code for generating the run.sh for dotnet specifically https://github.com/microsoft/Oryx/tree/main/src/startupscriptgenerator/src/dotnetcore

Reply:
Thanks Wali, but I was asking for the dotnet `run.sh` of the file you plan on appending this to. We'll have to understand what we are appending to. This will help us determine whether we can append to the end or not. 

The other point that should be addressed is whether we want to append to everything or not.  Do we know this? IMO we should only add to DotNet if we don't need this for the other platforms.

Reply:
Answering this question-" we want to append to everything or not?": Based on the discussion I had with Tulika we should append everything whatever user specifies as the run command on the `appsvc.yaml` file.

Reply:
I will generate the dotnet run.sh file and show it to you.